clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeliverWgsCramExecutor.scala

```diff
@@ -85,6 +74,11 @@ class DeliverWgsCramExecutor(deliverCommand: DeliverWgsCram)
       (cramData.cramMd5, cramData.cramPath) match {
         case (Some(cramMd5), Some(cramPath)) => {
 
+          /*
+           * FIXME: If we enable the google-cloud-nio adapter in the client,
```

Comment:
The `Files.delete()` call below should be in a `finally` clause in case any of the operations between `create` and `delete` throw.